common/main/java/com/couchbase/lite/internal/core/C4Log.java

```diff
@@ -86,10 +142,82 @@ public final void forceCallbackLevel(@NonNull LogLevel logLevel) {
         setCoreCallbackLevel();
     }
 
+    @VisibleForTesting
+    // This, apparently, should be the inverse of LOGGING_DOMAINS_FROM_C4
+    public final void setC4LogLevel(@NonNull EnumSet<LogDomain> domains, @NonNull LogLevel level) {
```

Comment:
I think this doesn't need to be public, anymore

Reply:
support.Log calls this method in initLogging()

`    @VisibleForTesting
    public static void initLogging() {
        C4Log.get().forceCallbackLevel(Database.log.getConsole().getLevel());
        C4Log.get().setC4LogLevel(LogDomain.ALL_DOMAINS, LogLevel.DEBUG);
    }
`

Reply:
support.Log calls this method in initLogging()

`    @VisibleForTesting
    public static void initLogging() {
        C4Log.get().forceCallbackLevel(Database.log.getConsole().getLevel());
        C4Log.get().setC4LogLevel(LogDomain.ALL_DOMAINS, LogLevel.DEBUG);
    }
`

Reply:
Then it isn't visible for testing, is it?